docs/core/tools/csproj.md

```diff
@@ -264,3 +264,37 @@ Base path for the *.nuspec* file.
 
 ### NuspecProperties
 Semicolon separated list of key=value pairs.
+
+## AssemblyInfo properties
+[Assembly attributes](../../framework/app-domains/set-assembly-attributes.md) that were typically present in an *AssemblyInfo* file are now automatically generated from properties.
+
+### Properties per attribute
+
+Each attribute has a property that control it's content and another to disable it's generation as follow :
```

Comment:
it's content -> its content
it's generation -> its generation
as follow : -> as shown in the following table: